kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestContext.java

```diff
@@ -36,7 +37,7 @@
     return getProducerPool().getProducer();
   }
 
-  default SchemaRegistryClient getSchemaRegistryClient() {
+  default Optional<SchemaRegistryClient> getSchemaRegistryClient() {
```

Comment:
We can't change this interface, or we will otherwise break `ce-kafka-rest` and `confluent-security-plugins`. Just use `@Nullable` and `null` instead of `Optional`. (This comes from someone who hates `null`)